test/e2e/storage/testsuites/driveroperations.go

```diff
@@ -71,8 +72,8 @@ func GetStorageClass(
 			Kind: "StorageClass",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			// Name must be unique, so let's base it on namespace name
-			Name: ns + "-" + suffix,
+			// Name must be unique, so let's base it on namespace name and random string
+			Name: ns + "-" + rand.String(4) + "-" + suffix,
```

Comment:
prefer  `GenerateName`

Reply:
@wongma7 @msau42 

Changed codes as below and squashed the commit, now sc is named like below:

PTAL

- sc naming
```
STEP: creating a StorageClass multivolume-4412-nfs-scndbcj 
```

- code change
<details>

```
# git diff
diff --git a/test/e2e/storage/testsuites/driveroperations.go b/test/e2e/storage/testsuites/driveroperations.go
index c0e30c3979..4d4298e61a 100644
--- a/test/e2e/storage/testsuites/driveroperations.go
+++ b/test/e2e/storage/testsuites/driveroperations.go
@@ -22,7 +22,7 @@ import (
        storagev1 "k8s.io/api/storage/v1"
        metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
        "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-       "k8s.io/apimachinery/pkg/util/rand"
+       "k8s.io/apiserver/pkg/storage/names"
        "k8s.io/kubernetes/test/e2e/framework"
        "k8s.io/kubernetes/test/e2e/storage/testpatterns"
 )
@@ -72,8 +72,8 @@ func GetStorageClass(
                        Kind: "StorageClass",
                },
                ObjectMeta: metav1.ObjectMeta{
-                       // Name must be unique, so let's base it on namespace name and random string
-                       Name: ns + "-" + rand.String(4) + "-" + suffix,
+                       // Name must be unique, so let's base it on namespace name and use GenerateName
+                       Name: names.SimpleNameGenerator.GenerateName(ns + "-" + suffix),
                },
                Provisioner:       provisioner,
                Parameters:        parameters,
```
</details>